composer.json

```diff
@@ -5,7 +5,7 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.1",
-    "spryker-sdk/spryk": "^0.1.0 || ^0.2.0",
+    "spryker-sdk/spryk": "^0.1.0 || ^0.2.0 || ^0.3.0",
```

Comment:
Why is this a major when you do allowing patch adjustments?
Either you make this a patch, or you remove the previous and only allow 0.3 from here on.

The latter makes more sense since this service part only works with the last of Spryk, the others are a lie anyway.